### ISSUE TITLE
Buttons alignment and order

### ISSUE THREAD

**Issue by felipeelia** [felipeelia/cf7-repeatable-fields#20]
Let's create two options:
1. Buttons alignment (left or right)
2. Buttons order (+/- or -/+)

**Comment by rtpHarry** [comment]
Until this is implemented you can swap the buttons by calling this in the document ready function:

```
    function swapButtons() {
        $(".wpcf7-field-group-add").each(function(index) {
            $(this).after($(this).prev().detach());
        });
    }
```

By using `detach` it keeps all of the events like the click handlers.
